.circleci/config.yml

```diff
@@ -9,6 +9,22 @@ jobs:
           name: Test
           command: |
             source $HOME/.bash_profile
+            pip install -e .
```

Comment:
@DustinKLo: once this is in the `develop` branch and hysds_commons has merged `develop-es7` into `develop`, can we remove this block entirely?

Reply:
sounds good

Reply:
i'll merge the PR, then create another PR with the new `.circleci` file